app/components/NavigationTab/NavigationLink.js

```diff
@@ -12,9 +12,16 @@ type Props = {
 };
 
 const NavigationLink = (props: Props) => {
+  // Custom isActive function as NavLink does not compare query params by default
+  const isActive = (match, location) => {
```

Comment:
Very smart! I noticed this was any issue back in #2782, but somehow forgot about it 🙃